reference/constraints/CardScheme.rst

```diff
@@ -41,6 +41,22 @@ on an object that will contain a credit card number.
             protected $cardNumber;
         }
 
+    .. code-block:: php-attributes
+
+        // src/Entity/Transaction.php
+        namespace App\Entity;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class Transaction
+        {
+            #[Assert\CardScheme(
+                schemes: ["VISA"],
+                message: "Your credit card number is invalid."
```

Comment:
```suggestion
                schemes: [Assert\CardScheme::VISA],
                message: 'Your credit card number is invalid.',
```

* We have added constants for the available card schemes and I suggest we make use of them.
* CS: prefer single quotes.
* CS: use trailing comma in multiline constructor calls.

Reply:
I added commas in all multiline constructor calls.